### PR TITLE
Identity linking with aad sso nodejs

### PR DESCRIPTION
**Pull request by NikitaJain-MSFT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -311,6 +312,8 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 [peoplepickeronadaptivecard#js]:samples/bot-people-picker-adaptive-card/nodejs
 [peoplepickeronadaptivecard#csharp]:samples/bot-people-picker-adaptive-card/csharp
 [graph-activity-feed-broadcast#js]:samples/graph-activity-feed-broadcast/nodejs
+[identity-link-with-sso#nodejs]:samples/identity-link-with-sso/nodejs
```
Comment (AbhijeetThakur-msft) [inline README.md:315]: Let's change the folder name as per the capability, here we can use app- as we are using multiple capabilities.

File: samples/identity-linking-with-sso/nodejs/README.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+- nodejs
+- javascript
+extensions:
+contentType: samples
+createdDate: "7-03-2022 00:15:13"
+---
+
+# Identity linking with sso
+
+This sample demos linking aad id with  facebook and google account of user in bot, ME and tab.
```
Comment (AbhijeetThakur-msft) [inline samples/identity-linking-with-sso/nodejs/README.md:18]: This sample demos linking user's AAD id with Facebook and google account of user from bot, ME and tab.

File: samples/identity-linking-with-sso/nodejs/README.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    Set a redirect URI:
+    * Select **Add a platform**.
+    * Select **web**.
+    * Enter the **redirect URI** for the app in the following format: `https://{Base_Url}/auth-end`. This will be the page where a successful implicit grant flow will redirect the user.
+    Enable implicit grant by checking the following boxes:  
+    ✔ ID Token  
+    ✔ Access Token  
+14.  Navigate to the **Certificates & secrets**. In the Client secrets section, click on "+ New client secret". Add a description(Name of the secret) for the secret and select “Never” for Expires. Click "Add". Once the client secret is created, copy its value, it need to be placed in the appsettings.json.
+
+15. Create a Bot Registration
```
Comment (AbhijeetThakur-msft) [inline samples/identity-linking-with-sso/nodejs/README.md:101]: Let's change this to Azure Bot Service

File: samples/identity-linking-with-sso/nodejs/README.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+   In Azure portal, create a [Bot Framework registration resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2).
```
Comment (AbhijeetThakur-msft) [inline samples/identity-linking-with-sso/nodejs/README.md:102]: Please change this to Azure Bot service.

**Review by AbhijeetThakur-msft — CHANGES_REQUESTED** [review]

**Review by AbhijeetThakur-msft — APPROVED** [review]

**Review by v-chetsh — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**